cmd/run.go

```diff
@@ -59,6 +59,9 @@ func SetupRunFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().Bool("regtest", false,
 		"Can only be used in conjunction with --testnet. Creates a private testnet node with fast block times"+
 			"and instantly spendable block rewards.")
+	cmd.PersistentFlags().String("postgres-uri", "", "URI for syncing the node with postgres")
+	cmd.PersistentFlags().String("postgres-data-dir", "", "Start a local embedded postgres instance"+
+		"and persist its data in this directory")
```

Comment:
I would clarify that it is starting a local embedded postgres instance *instead* of a remote postgres instance and that it will be ignored if a URI is provided.  I think these info strings make sense but making it a touch more explicit would be nice.

Side note: it would be nice to clarify our node's current working relationship with postgres somewhere.  That is, if you spin up a postgres node, it will use postgres for most things and badger DB for some things that have not yet been migrated (with examples).  I'm not sure the best place to make that clear though.  Maybe we could put something here and in the docs?